data-raw/babies.R

```diff
@@ -0,0 +1,28 @@
+## code to prepare `babies` dataset goes here
+
+babies <- readr::read_csv("https://wjhopper.github.io/SDS-201/data/babies.csv")
```

Comment:
You don't need to call `readr::` explicitly here. Also please pipe the URL to `read_csv()`.